lib/helpers.js

```diff
@@ -141,35 +157,6 @@ exports.checkRelationName = function(modelDefinition, name) {
 };
 
 /**
- * Get the root command name based on the env var
- * @returns {string}
- */
-exports.getCommandName = function() {
```

Comment:
Is there any particular reason for moving the implementation of this function? Can we keep it in the old place, to reduce the patch size?